controllers/snapshot/snapshot_adapter.go

```diff
@@ -463,6 +494,12 @@ func (a *Adapter) EnsureSnapshotEnvironmentBindingExist() (controller.OperationR
 
 		}
 	}
+	// Get the start time when AppStudioTests have finished
```

Comment:
This is not the right place for this metric, it cares about release not SEBs, and also this metric shouldn't be duplicated, I see it on 2 places already